tests/models/test_horovod.py

```diff
@@ -139,11 +142,35 @@ def test_horovod_amp(tmpdir):
         gpus=2,
         deterministic=True,
         distributed_backend='horovod',
+        amp_backend='apex',
```

Comment:
it means that Horovod as we have it now does not work with AMP from PT 1.6, correct?

Reply:
Yes, it doesn't work with torch.cuda.amp

Reply:
Can confirm, this is just an issue with Horovod.